CleanSwift/CleanSwift/MainScreen/MainInteractor.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  MainInteractor.swift
+//  CleanSwift
+//
+//  Created by Валентин Коскин on 15.02.2023.
+//
+
+import UIKit
```

Comment:
```suggestion
import UIKit
```
Не уверен что интерактор может работать юай элементами